applicationset/generators/pull_request.go

```diff
@@ -67,10 +72,15 @@ func (g *PullRequestGenerator) GenerateParams(appSetGenerator *argoprojiov1alpha
 	}
 
 	pulls, err := pullrequest.ListPullRequests(ctx, svc, appSetGenerator.PullRequest.Filters)
+	params := make([]map[string]any, 0, len(pulls))
 	if err != nil {
+		if pullrequest.IsRepositoryNotFoundError(err) && g.GetContinueOnRepoNotFoundError(appSetGenerator) {
+			log.WithError(err).WithField("generator", g).
+				Warn("Skipping params generation for this repository since it was not found.")
+			return params, nil
```

Comment:
```suggestion
			return params, nil
```
I'm thinking if it's correct to return the `params` here. If you get an error while listing Pull Requests, chances are the `params` value can be wrong in that case.

Reply:
The params in the case when there was also a not-found error are just an empty list.